### PR TITLE
Docs: Add vaul.token.create_root metric

### PR DESCRIPTION
**Pull request by michelvocks**
Fixes https://github.com/hashicorp/vault/issues/7974

### CODE REVIEW

File: website/source/docs/internals/telemetry.html.md [hunk 1]
```diff
@@ -138,6 +138,7 @@ These operations take a request and response with an associated lease and regist
 | `vault.policy.delete_policy` | Time taken to delete a policy | ms | summary |
 | `vault.policy.set_policy` | Time taken to set a policy | ms | summary |
 | `vault.token.create` | The time taken to create a token | ms | summary |
+| `vault.token.create_root` | Number of created root tokens | token | gauge |
```
Comment (ncabatoff) [inline website/source/docs/internals/telemetry.html.md]: Should be counter, not gauge.

Comment (michelvocks) [reply website/source/docs/internals/telemetry.html.md]:
```suggestion
| `vault.token.create_root` | Number of created root tokens | token | counter |
```

Comment (michelvocks) [reply website/source/docs/internals/telemetry.html.md]: Whops, right. Thanks!

**Comment by michelvocks** [comment]
@ncabatoff Agreed. What do you think about this?

**Pull request merged**